database/query.go

```diff
@@ -7,6 +7,7 @@ var (
 		"name varchar (150), " +
 		"available integer, " +
 		"capacity integer, " +
+		"status text, " +
```

Comment:
Convert it to integer would be easier to manage i guess. for example we can this status:
1 - Running
0 - Suspended